src/constants.js

```diff
@@ -6,5 +6,8 @@ export default {
     copy: {
```

Comment:
Do we know it's "signal", specifically? Otherwise, yes, the app needs some bare bones messages for such circumstances.